app/views/reservations/_js_variables.html.haml

```diff
@@ -1,5 +1,6 @@
 :javascript
   var events_path     = "#{facility_instrument_reservations_path(@instrument.facility, @instrument, format: 'js', with_details: @instrument.show_details?)}";
+  var orders_path_base = "#{facility_orders_path(@instrument.facility)}";
```

Comment:
I think we don't want it here. The "Edit" link should not appear on the user facing views (new/edit reservation), as it will 404 for non-facility staff/admins. Where else is this partial shared?

Reply:
Gotcha! That’s used from `reservations/new` and `reservations/edit`, so I’ve removed it in e2989374c. `orders_path_base` is now defined in a couple of views under `facility_reservations`, and in `instruments/schedule`, which are all places that I think we want this, right?